OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -93,6 +91,7 @@ public static void Configure(this WebApplication app)
                 Predicate = healthCheck => healthCheck.Tags.Contains("readiness"),
                 AllowCachingResponses = false,
             })
+            .RequireHost("*:9000")
```

Comment:
(Code style) Magic number, that probably has to be `ApplicationPorts:HealthPort`.

Reply:
Somehow missed that, it's fixed now